src/app/_css/globals.css

```diff
@@ -0,0 +1,86 @@
+:root {
+  /* Colors */
+
+  --sub-blue-900: #2952DD;
+  --sub-blue-800: #045CFC;
+  --sub-blue-700: #006FFF;
+  --sub-blue-600: #0083FF;
+  --sub-blue-500: #0093FF;
+  --sub-blue-400: #00A4FF;
+  --sub-blue-300: #3CB6FF;
+  --sub-blue-200: #84CBFF;
+  --sub-blue-100: #B8DEFF;
+  --sub-blue-50: #E1F2FF;
+
+  --sub-purple-900: #0000CD;
+  --sub-purple-800: #0000DB;
+  --sub-purple-700: #2E00E3;
+  --sub-purple-600: #4801EC;
+  --sub-purple-500: #5807F2;
+  --sub-purple-400: #773BFF;
+  --sub-purple-300: #9563FF;
+  --sub-purple-200: #B693FE;
+  --sub-purple-100: #D4BEFE;
+  --sub-purple-50: #EFE5FF;
+
+  --dark-rock-800: #403F4C;
+  --soft-white-100: #F6F6F6;
+
+  /* Typography */
+  --colfax-medium: "Colfax Medium", Roboto, sans-serif;
+  --colfax-regular: Colfax, Roboto, sans-serif;
+  /* TODO: Replace placeholder TNR for actual Acta*/
+  --acta-book: "Acta Book", "Times New Roman", serif;
+
+
+  h1 {
+    font-size: 56px;
```

Comment:
This approach is valid, though it means you will need to always use the correct html elements to get the intended font sizes. Usually we decouple this, and create something like
```
`--font-size-56`: 56px

h1 {
  font-size: var(--font-size-56)
}
```
Then we can use the --font-size-56 variable anywhere else if needed.